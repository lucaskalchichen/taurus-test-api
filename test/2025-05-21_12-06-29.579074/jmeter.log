2025-05-21 12:06:38,184 INFO o.a.j.u.JMeterUtils: Setting Locale to en
2025-05-21 12:06:38,195 INFO o.a.j.JMeter: Loading user properties from: /home/lucas/.bzt/jmeter-taurus/5.5/bin/user.properties
2025-05-21 12:06:38,195 INFO o.a.j.JMeter: Loading system properties from: /home/lucas/.bzt/jmeter-taurus/5.5/bin/system.properties
2025-05-21 12:06:38,196 INFO o.a.j.JMeter: Loading additional properties from: /mnt/d/Universidad/recursos/taurus api/test/2025-05-21_12-06-29.579074/jmeter-bzt.properties
2025-05-21 12:06:38,197 INFO o.a.j.JMeter: Setting System properties from file: /mnt/d/Universidad/recursos/taurus api/test/2025-05-21_12-06-29.579074/system.properties
2025-05-21 12:06:38,200 INFO o.a.j.JMeter: Copyright (c) 1998-2022 The Apache Software Foundation
2025-05-21 12:06:38,200 INFO o.a.j.JMeter: Version 5.5
2025-05-21 12:06:38,200 INFO o.a.j.JMeter: java.version=11.0.27
2025-05-21 12:06:38,200 INFO o.a.j.JMeter: java.vm.name=OpenJDK 64-Bit Server VM
2025-05-21 12:06:38,200 INFO o.a.j.JMeter: os.name=Linux
2025-05-21 12:06:38,200 INFO o.a.j.JMeter: os.arch=amd64
2025-05-21 12:06:38,200 INFO o.a.j.JMeter: os.version=5.15.167.4-microsoft-standard-WSL2
2025-05-21 12:06:38,201 INFO o.a.j.JMeter: file.encoding=UTF-8
2025-05-21 12:06:38,201 INFO o.a.j.JMeter: java.awt.headless=true
2025-05-21 12:06:38,201 INFO o.a.j.JMeter: Max memory     =4160749568
2025-05-21 12:06:38,201 INFO o.a.j.JMeter: Available Processors =28
2025-05-21 12:06:38,203 INFO o.a.j.JMeter: Default Locale=English
2025-05-21 12:06:38,204 INFO o.a.j.JMeter: JMeter  Locale=English
2025-05-21 12:06:38,204 INFO o.a.j.JMeter: JMeterHome=/home/lucas/.bzt/jmeter-taurus/5.5
2025-05-21 12:06:38,204 INFO o.a.j.JMeter: user.dir  =/mnt/d/Universidad/recursos/taurus api/test
2025-05-21 12:06:38,204 INFO o.a.j.JMeter: PWD       =/mnt/d/Universidad/recursos/taurus api/test
2025-05-21 12:06:38,210 INFO o.a.j.JMeter: IP: 127.0.1.1 Name: kalchi-pc FullName: kalchi-pc.
2025-05-21 12:06:38,210 INFO o.a.j.JMeter: user.classpath=/mnt/d/Universidad/recursos/taurus api/test/2025-05-21_12-06-29.579074:/mnt/d/Universidad/recursos/taurus api/test/2025-05-21_12-06-29.579074
2025-05-21 12:06:38,210 INFO o.a.j.JMeter: Adding to classpath and loader: /mnt/d/Universidad/recursos/taurus api/test/2025-05-21_12-06-29.579074
2025-05-21 12:06:38,211 INFO o.a.j.JMeter: Adding to classpath and loader: /mnt/d/Universidad/recursos/taurus api/test/2025-05-21_12-06-29.579074
2025-05-21 12:06:38,218 INFO o.a.j.s.FileServer: Default base='/mnt/d/Universidad/recursos/taurus api/test'
2025-05-21 12:06:38,218 INFO o.a.j.s.FileServer: Set new base='/mnt/d/Universidad/recursos/taurus api/test/2025-05-21_12-06-29.579074'
2025-05-21 12:06:38,298 INFO o.a.j.s.SaveService: Testplan (JMX) version: 2.2. Testlog (JTL) version: 2.2
2025-05-21 12:06:38,309 INFO o.a.j.s.SaveService: Using SaveService properties version 5.0
2025-05-21 12:06:38,311 INFO o.a.j.s.SaveService: Using SaveService properties file encoding UTF-8
2025-05-21 12:06:38,312 INFO o.a.j.s.SaveService: Loading file: /mnt/d/Universidad/recursos/taurus api/test/2025-05-21_12-06-29.579074/modified_requests.jmx
2025-05-21 12:06:38,329 INFO o.a.j.p.h.c.CacheManager: Will only cache the following methods: [GET]
2025-05-21 12:06:38,329 INFO o.a.j.p.h.c.CookieManager: Settings: Delete null: true Check: true Allow variable: true Save: false Prefix: COOKIE_
2025-05-21 12:06:38,336 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for text/html is org.apache.jmeter.protocol.http.parser.LagartoBasedHtmlParser
2025-05-21 12:06:38,336 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for application/xhtml+xml is org.apache.jmeter.protocol.http.parser.LagartoBasedHtmlParser
2025-05-21 12:06:38,336 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for application/xml is org.apache.jmeter.protocol.http.parser.LagartoBasedHtmlParser
2025-05-21 12:06:38,336 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for text/xml is org.apache.jmeter.protocol.http.parser.LagartoBasedHtmlParser
2025-05-21 12:06:38,337 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for text/vnd.wap.wml is org.apache.jmeter.protocol.http.parser.RegexpHTMLParser
2025-05-21 12:06:38,337 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for text/css is org.apache.jmeter.protocol.http.parser.CssParser
2025-05-21 12:06:38,353 INFO o.a.j.e.StandardJMeterEngine: Running the test!
2025-05-21 12:06:38,353 INFO o.a.j.s.SampleEvent: List of sample_variables: []
2025-05-21 12:06:38,354 INFO o.a.j.s.SampleEvent: List of sample_variables: []
2025-05-21 12:06:38,359 INFO o.a.j.e.u.CompoundVariable: Note: Function class names must contain the string: '.functions.'
2025-05-21 12:06:38,359 INFO o.a.j.e.u.CompoundVariable: Note: Function class names must not contain the string: '.gui.'
2025-05-21 12:06:38,491 INFO o.a.j.JMeter: Running test (1747839998491)
2025-05-21 12:06:38,499 INFO o.a.j.e.StandardJMeterEngine: Starting ThreadGroup: 1 : api-test
2025-05-21 12:06:38,499 INFO o.a.j.e.StandardJMeterEngine: Starting 42 threads for group api-test.
2025-05-21 12:06:38,500 INFO o.a.j.e.StandardJMeterEngine: Thread will continue on error
2025-05-21 12:06:38,500 INFO o.a.j.t.ThreadGroup: Starting thread group... number=1 threads=42 ramp-up=3 delayedStart=false
2025-05-21 12:06:38,505 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-1
2025-05-21 12:06:38,517 INFO o.a.j.t.ThreadGroup: Started thread group number 1
2025-05-21 12:06:38,517 INFO o.a.j.e.StandardJMeterEngine: All thread groups have been started
2025-05-21 12:06:38,517 INFO o.a.j.p.h.s.HTTPHCAbstractImpl: Local host = kalchi-pc
2025-05-21 12:06:38,519 INFO o.a.j.p.h.s.HTTPHC4Impl: HTTP request retry count = 0
2025-05-21 12:06:38,520 INFO o.a.j.s.SampleResult: Note: Sample TimeStamps are START times
2025-05-21 12:06:38,520 INFO o.a.j.s.SampleResult: sampleresult.default.encoding is set to UTF-8
2025-05-21 12:06:38,520 INFO o.a.j.s.SampleResult: sampleresult.useNanoTime=true
2025-05-21 12:06:38,520 INFO o.a.j.s.SampleResult: sampleresult.nanoThreadSleep=5000
2025-05-21 12:06:38,574 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-2
2025-05-21 12:06:38,644 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-3
2025-05-21 12:06:38,716 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-4
2025-05-21 12:06:38,787 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-5
2025-05-21 12:06:38,857 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-6
2025-05-21 12:06:38,927 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-7
2025-05-21 12:06:38,997 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-8
2025-05-21 12:06:39,069 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-9
2025-05-21 12:06:39,140 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-10
2025-05-21 12:06:39,210 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-11
2025-05-21 12:06:39,282 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-12
2025-05-21 12:06:39,353 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-13
2025-05-21 12:06:39,425 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-14
2025-05-21 12:06:39,494 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-15
2025-05-21 12:06:39,566 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-16
2025-05-21 12:06:39,637 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-17
2025-05-21 12:06:39,708 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-18
2025-05-21 12:06:39,779 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-19
2025-05-21 12:06:39,849 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-20
2025-05-21 12:06:39,921 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-21
2025-05-21 12:06:39,992 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-22
2025-05-21 12:06:40,062 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-23
2025-05-21 12:06:40,133 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-24
2025-05-21 12:06:40,204 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-25
2025-05-21 12:06:40,276 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-26
2025-05-21 12:06:40,346 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-27
2025-05-21 12:06:40,417 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-28
2025-05-21 12:06:40,488 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-29
2025-05-21 12:06:40,560 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-30
2025-05-21 12:06:40,632 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-31
2025-05-21 12:06:40,701 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-32
2025-05-21 12:06:40,773 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-33
2025-05-21 12:06:40,843 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-34
2025-05-21 12:06:40,914 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-35
2025-05-21 12:06:40,989 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-36
2025-05-21 12:06:41,056 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-37
2025-05-21 12:06:41,127 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-38
2025-05-21 12:06:41,199 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-39
2025-05-21 12:06:41,270 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-40
2025-05-21 12:06:41,340 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-41
2025-05-21 12:06:41,412 INFO o.a.j.t.JMeterThread: Thread started: api-test 1-42
2025-05-21 12:06:46,876 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-1
2025-05-21 12:06:46,877 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-1
2025-05-21 12:06:50,523 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-2
2025-05-21 12:06:50,524 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-2
2025-05-21 12:06:53,700 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-3
2025-05-21 12:06:53,701 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-3
2025-05-21 12:06:54,756 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-4
2025-05-21 12:06:54,757 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-4
2025-05-21 12:06:56,382 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-5
2025-05-21 12:06:56,382 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-5
2025-05-21 12:07:00,150 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-6
2025-05-21 12:07:00,150 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-6
2025-05-21 12:07:03,742 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-8
2025-05-21 12:07:03,742 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-8
2025-05-21 12:07:03,742 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-7
2025-05-21 12:07:03,743 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-7
2025-05-21 12:07:06,931 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-9
2025-05-21 12:07:06,932 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-9
2025-05-21 12:07:07,624 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-11
2025-05-21 12:07:07,625 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-11
2025-05-21 12:07:08,711 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-10
2025-05-21 12:07:08,711 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-10
2025-05-21 12:07:10,069 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-12
2025-05-21 12:07:10,069 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-12
2025-05-21 12:07:12,003 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-14
2025-05-21 12:07:12,005 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-14
2025-05-21 12:07:12,879 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-13
2025-05-21 12:07:12,881 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-13
2025-05-21 12:07:15,235 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-18
2025-05-21 12:07:15,235 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-18
2025-05-21 12:07:16,403 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-16
2025-05-21 12:07:16,403 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-16
2025-05-21 12:07:17,058 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-15
2025-05-21 12:07:17,058 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-15
2025-05-21 12:07:18,378 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-17
2025-05-21 12:07:18,379 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-17
2025-05-21 12:07:21,411 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-22
2025-05-21 12:07:21,431 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-22
2025-05-21 12:07:21,429 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-19
2025-05-21 12:07:21,431 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-21
2025-05-21 12:07:21,431 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-21
2025-05-21 12:07:21,431 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-20
2025-05-21 12:07:21,431 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-19
2025-05-21 12:07:21,432 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-20
2025-05-21 12:07:23,962 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-24
2025-05-21 12:07:23,962 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-24
2025-05-21 12:07:24,658 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-25
2025-05-21 12:07:24,662 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-25
2025-05-21 12:07:26,469 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-27
2025-05-21 12:07:26,470 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-27
2025-05-21 12:07:26,605 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-23
2025-05-21 12:07:26,609 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-23
2025-05-21 12:07:27,432 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-29
2025-05-21 12:07:27,432 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-29
2025-05-21 12:07:28,238 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-26
2025-05-21 12:07:28,238 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-26
2025-05-21 12:07:29,680 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-28
2025-05-21 12:07:29,680 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-28
2025-05-21 12:07:29,680 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-30
2025-05-21 12:07:29,681 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-30
2025-05-21 12:07:32,726 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-33
2025-05-21 12:07:32,726 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-32
2025-05-21 12:07:32,726 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-31
2025-05-21 12:07:32,726 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-33
2025-05-21 12:07:32,726 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-32
2025-05-21 12:07:32,726 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-31
2025-05-21 12:07:32,726 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-35
2025-05-21 12:07:32,727 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-35
2025-05-21 12:07:33,478 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-34
2025-05-21 12:07:33,478 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-34
2025-05-21 12:07:34,258 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-36
2025-05-21 12:07:34,258 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-36
2025-05-21 12:07:35,313 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-37
2025-05-21 12:07:35,314 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-37
2025-05-21 12:07:37,462 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-39
2025-05-21 12:07:37,462 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-39
2025-05-21 12:07:37,462 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-38
2025-05-21 12:07:37,463 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-38
2025-05-21 12:07:37,463 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-40
2025-05-21 12:07:37,463 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-40
2025-05-21 12:07:38,158 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-42
2025-05-21 12:07:38,158 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-42
2025-05-21 12:07:38,841 INFO o.a.j.t.JMeterThread: Thread is done: api-test 1-41
2025-05-21 12:07:38,842 INFO o.a.j.t.JMeterThread: Thread finished: api-test 1-41
2025-05-21 12:07:38,842 INFO o.a.j.e.StandardJMeterEngine: Notifying test listeners of end of test
2025-05-21 12:07:38,844 INFO o.a.j.e.StandardJMeterEngine: Forced JVM shutdown requested at end of test
