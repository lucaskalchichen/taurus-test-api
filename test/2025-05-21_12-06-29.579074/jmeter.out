Created the tree successfully using /mnt/d/Universidad/recursos/taurus api/test/2025-05-21_12-06-29.579074/modified_requests.jmx
Starting standalone test @ 2025 May 21 12:06:38 ART (1747839998348)
Waiting for possible Shutdown/StopTestNow/HeapDump/ThreadDump message on port 4445
Tidying up ...    @ 2025 May 21 12:07:38 ART (1747840058843)
... end of run
